### PR TITLE
Feature/add classifiers import

### PR DESCRIPTION
**Pull request by a-solovyev12**
This PR adds scripts for import of pre-trained classifiers and corresponding SDRF files from FTP server.

### CODE REVIEW

File: import_sdrf_files.R [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+suppressPackageStartupMessages(require(RCurl))
+
+# parse config file or use default values
+if(!is.na(opt$config_file)){
+    config = yaml.load_file(opt$config_file)
+    datasets = toupper(config$datasets)
+    scxa_experiments_prefix = config$scxa_experiments_prefix
+    if(!endsWith(scxa_experiments_prefix, "/")) scxa_experiments_prefix = paste(scxa_experiments_prefix, "/", sep="")
+} else {
+    scxa_experiments_prefix = "ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/atlas/sc_experiments/"
```
Comment (pinin4fjords) [inline import_sdrf_files.R]: Could you put the default URIs in some single config file so they're not hard-coded in any script? I'm seeing the same path in a few places which is a recipe for maintenance issues ;-)

File: import_classifiers.R [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            help = "Path for directory storing imported classifiers"
+    )
+)
+
+opt = wsc_parse_args(option_list, mandatory = c("tool", "classifiers_output_dir"))
+# import dependencies
+suppressPackageStartupMessages(require(optparse))
+suppressPackageStartupMessages(require(workflowscriptscommon))
+suppressPackageStartupMessages(require(R.utils))
+suppressPackageStartupMessages(require(yaml))
```
Comment (pinin4fjords) [inline import_classifiers.R]: Probably move the yaml load to the condition below- you don't need it if you don't need to load the yaml

File: import_sdrf_files.R (hunk 1 above)
Comment (a-solovyev12) [reply import_sdrf_files.R]: @pinin4fjords but there is already an option - either use user-provided prefixes from yaml config or use default ones

**Review by pinin4fjords — APPROVED** [review]
LGTM except for the minor comments.

File: import_sdrf_files.R (hunk 1 above)
Comment (rubenchazarra) [reply import_sdrf_files.R]: Minor doubt @a-solovyev12, what's the difference between `action = "store"` and ` action = "store_true"` ?

**Review by rubenchazarra — APPROVED** [review]
LGTM!

File: import_sdrf_files.R (hunk 1 above)
Comment (a-solovyev12) [reply import_sdrf_files.R]: @rubenchazarra, `action=store_true` is used specifically for Boolean variables.

**Review by pinin4fjords — APPROVED** [review]
Nice- thanks for addressing comments @a-solovyev12 , LGTM

**Comment by a-solovyev12** [comment]
thanks for review @pinin4fjords !

**Pull request merged**